controller/schema.mustache

```diff
@@ -1,5 +1,7 @@
 {
   "type": "object",
+  "name": "{{name}}",
```

Comment:
to quote Dave "Yeah you can put whatever you damn well like in there! It's the wild west."